kmm/kmm-feed-test-fixtures/src/commonMain/kotlin/com/gchristov/newsfeed/kmmfeedtestfixtures/FakeFeedRepository.kt

```diff
@@ -16,6 +16,7 @@ class FakeFeedRepository(
 
     private var _cacheCleared = false
     private var _pageIndex = 0
+    private var _lastSearchQuery = mutableMapOf<String, String>()
```

Comment:
No need for a map here. Can use a string parameter to store instead.

Reply:
Do you mean is it fine with just one local, private variable `_lastSearchQuery` ? It's possible, depends on the tests written (none a this point)

Reply:
Done with local variable

Reply:
Yep exactly what I meant 👍 The map was used in `FakePostRepository` because we're storing `favourite` state per post so the efficient way of doing that is with a map. In our case here we only have one last search query so a variable is okay.